infra/base-images/base-runner/bad_build_check

```diff
@@ -95,6 +95,10 @@ function check_startup_crash {
   if [[ "$FUZZING_ENGINE" = libfuzzer ]]; then
     $FUZZER -runs=$MIN_NUMBER_OF_RUNS &>$FUZZER_OUTPUT
     CHECK_PASSED=$(egrep "Done $MIN_NUMBER_OF_RUNS runs" -c $FUZZER_OUTPUT)
+    if [[ "$CHECK_PASSED" -ne "0" && "${RUN_MINIJAIL:-no}" = yes ]]; then
```

Comment:
I don't think we should even run without minijail when the developer explicitly provided the option, so can change to just an if-else.